docs/tuning.rst

```diff
@@ -350,6 +386,14 @@ minimal set of torrents you are actually interested in.
 To get an update with only the torrents that have changed since last time, call
 ``session::post_torrent_updates()``.
 
+high latency networks
+=====================
+
```

Comment:
I would interpret "high latency" as latency / bandwidth. There are plenty of high-bandwidth networks where the latency poses a similar problem (as in the bandwidth delay product is higher than normal)